package.json

```
@@ -24,6 +24,7 @@
     "@fortawesome/fontawesome-free": "6.0.0",
     "bootstrap": "5.1.3",
     "directory-tree": "3.2.2",
+    "dotenv": "^16.0.0",
```

Oiya kelupaan, jangan lupa di pin dependensinya

```suggestion
    "dotenv": "16.0.0",
```